addressbase-local-custodian-codes-street.csv

```diff
@@ -0,0 +1,393 @@
+﻿Local Custodian Code,Authority,Region,Country

```

Comment:
Similarly, I think the CSVs can be added in their own self-contained commit. The commit would ideally describe how you created these CSVs too, or where you downloaded them from.

I also think they should be grouped somewhere, e.g. in a `data/` folder instead of at the root of the repository.

Reply:
Good point @ChrisBAshton, I have moved them to data folder in the separate commit.

Reply:
Thanks @baisa - much easier to review now!

It's worth adding a [link to the Trello card](https://trello.com/c/OQKm9z4V/2758-add-local-custodian-codes-to-local-links-manager-5) in the commit message itself, rather than just refer to "the card", as it saves a developer having to find the PR associated with the commit, to find the Trello card.

Reply:
@ChrisBAshton makes sense!